UI/CMakeLists.txt

```diff
@@ -39,6 +39,13 @@ else()
 	set(MIXER_ENABLED TRUE)
 endif()
 
+find_package(libcaffeine)
+if(LIBCAFFEINE_FOUND)
```

Comment:
will this overwrite CAFFEINE_ENABLED being set manually?

Reply:
Hmm it's not implemented as an option so that wasn't considered. I'll make that change

Reply:
I added a BUILD_CAFFEINE option

Reply:
Cool looks good.